roles/pulp-devel/tasks/add_fixtures.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Enable unprivileged_userns_clone
+  sysctl:
+    name: kernel.unprivileged_userns_clone
+    value: "1"
+  become: true
```

Comment:
This task should probably go with the podman installation, but I would like to hold out on doing this until https://github.com/pulp/ansible-pulp/pull/222 is through (since this refactors where the podman installation is).

Reply:
From my local test it looks like this will not work, and is not needed on Fedora.

Reply:
CentOS is also unhappy in a slightly different way. I will try to fix and test these issues locally.